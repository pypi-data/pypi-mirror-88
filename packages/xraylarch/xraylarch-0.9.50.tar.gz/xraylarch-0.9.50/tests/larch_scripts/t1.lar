#print '---- Larch test 1'
x = sqrt(11)
y = cos(x)
print 'x,y = ', x, y
n = 1
while n<8:
   print ' Loop', n
   n = n + 1
endwhile


