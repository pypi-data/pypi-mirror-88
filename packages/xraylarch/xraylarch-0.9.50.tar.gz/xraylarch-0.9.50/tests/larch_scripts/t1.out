x,y =  3.31662479036 -0.984720943124
 Loop 1
 Loop 2
 Loop 3
 Loop 4
 Loop 5
 Loop 6
 Loop 7
